test/manifests/manifests.go

```diff
@@ -1,28 +1,9 @@
 package manifests
```

Comment:
Do we still need to keep this file?

Reply:
I can imagine adding some other more focused operator-centric tests here later, didn't want to tear down all the machinery yet if it's benign.